.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+<!--
+Thanks for reporting issues back to PerfCascade!
+
+If there's something that doesn't work, please explain carefully how we can reproduce your problem and help us by checking the console log in your browser. If you got an error, please include it in your issue!
+

```

Comment:
Great idea! Perhaps we can add that it helps to upload the har that causes the issue?